### PR TITLE
Unify READMEs

### PR DESCRIPTION
**Pull request by 2can**
This PR is the 1st stage of unification of the structure of apps READMEs.
1. Common app header & image
2. Short description
3. Configuration / detailed description (if present)
4. Development (app template description linked to app template README, to give main focus on the app specific content)

### CODE REVIEW

File: apps/crm/README.md [hunk 1]
```diff
@@ -1,7 +1,146 @@
...
+- [Mailchimp](http://mailchimp.com/)
+- [Twilio Segment](https://segment.com/) [_coming soon_]
+- [RudderStack](https://www.rudderstack.com/) [_coming soon_]
+
+There are two main features CRM App provides:
+
+- Initial indexing of existing Saleor customers and sending them to the provider contact list
+- Synchronizing new customers between Saleor and the provider
+
+:::caution
```
Comment (peelar) [inline apps/crm/README.md:17]: Those blocks don't exist in Markdown. I suggest using `>`.

File: apps/crm/README.md [hunk 2]
```diff
@@ -1,7 +1,146 @@
...
+To successfully configure the CRM App, you must:
+
+- Have a Mailchimp account
+- Have a Saleor user with `MANAGE_APPS` permission
+
+:::
+
+## Installation
+
+The App can be installed from [Saleor App Store](developer/app-store/overview.mdx). To install the App, you must have `MANAGE_USERS` and `MANAGE_APPS` [permissions](developer/permissions.mdx#app-permissions).
```
Comment (peelar) [inline apps/crm/README.md:28]: The links won't work in the repository. Do we have a plan for addressing that?

**Review by peelar — CHANGES_REQUESTED** [review]

File: apps/crm/README.md (hunk 2 above)
Comment (lkostrowski) [reply apps/crm/README.md:28]: These must be absolute links

**Review by peelar — APPROVED** [review]

**Review by peelar — APPROVED** [review]

**Comment by lkostrowski** [comment]
@2can what is the status?

**Pull request closed without merging**